be/src/storage/tablet_schema.cpp

```diff
@@ -393,7 +394,18 @@ bool TabletColumn::is_format_v2_column() const {
  * TabletSchema
  ******************************************************************/
 
+TabletSchema::TabletSchema(const TabletSchemaPB& schema_pb) {
+    init_from_pb(schema_pb);
+    ExecEnv::GetInstance()->tablet_meta_mem_tracker()->consume(mem_usage());
```

Comment:
IMO, this is not a good idea to call ExecEnv in tablet_schame.cpp. This will make TabletSchema depend on ExecEnv.

Reply:
If you don’t do this, you need to store MemTracker in it to increase memory usage

Reply:
Any suggestions ?

Reply:
It is best to have a unified memory allocator to do these things, but currently some metadata is obtained from thrift desrialze, which can not be realized in a short time.

Reply:
```
        auto ptr = std::make_shared<const TabletSchema>(
                schema_pb, this, TabletSchema::ColumnDeleter(ExecEnv::GetInstance()->tablet_meta_mem_tracker()));
```

Reply:
Fixed